docs/SX.md

```diff
@@ -0,0 +1,467 @@
+---
+layout: default
+title: "SX"
+---
+
+# The `sx` prop
+
+All react-admin components expose an `sx` prop, which allows to customize the component style. This `sx` prop is kind of like [React's `style` prop](https://react.dev/reference/react-dom/components/common#applying-css-styles), but it's more powerful. It uses the CSS-in-JS solution offered by MUI, [MUI System](https://mui.com/system/basics/#the-sx-prop), powered by [emotion](https://emotion.sh/docs/introduction). 
```

Comment:
```suggestion
All react-admin components expose a `sx` prop, which allows to customize the component style. This `sx` prop is kind of like [React's `style` prop](https://react.dev/reference/react-dom/components/common#applying-css-styles), but it's more powerful. It uses the CSS-in-JS solution offered by MUI, [MUI System](https://mui.com/system/basics/#the-sx-prop), powered by [emotion](https://emotion.sh/docs/introduction). 
```

Reply:
No, my version is correct

https://www.quora.com/When-a-word-starts-with-an-%E2%80%9Cs%E2%80%9D-do-you-use-%E2%80%9Ca%E2%80%9D-or-%E2%80%9Can%E2%80%9D-before-the-word